ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -163,7 +157,7 @@ interface State {
 }
 
 
-export const AdminUser = withUrlParams()(class extends React.Component<Props, State> {
+export const AdminUser = withRouter(class extends React.Component<Props, State> {
```

Comment:
maybe we can provide a `withParams` HOC that behaves like the `useParams` hook? That way, the way we use the returned object is the same in both class components and hook components.